lm_eval/datasets/pile/pile.py

```diff
@@ -38,8 +38,8 @@
 
 _HOMEPAGE = "https://pile.eleuther.ai/"
 
-# TODO: Add the licence for the dataset here if you can find it
-_LICENSE = ""
+# License available at https://github.com/EleutherAI/the-pile/blob/master/LICENSE
+_LICENSE = "MIT License"
```

Comment:
This license is for the code to generate the Pile, not the dataset itself, which cannot be assigned a new license by us.